front/src/modules/rollingStock/consts.ts

```diff
@@ -18,7 +20,7 @@ export type Curve = {
   speeds: number[];
 };
 
-export type SelectedCurve = {
+export type SelectedCurves = {
```

Comment:
Use singular here as it is only 1 curve
```suggestion
export type SelectedCurve = {
```